lib/binary_protocol/serializers.js

```diff
@@ -305,7 +305,8 @@ class WriteMultipleSerializer extends Protocol {
     static serialize(dw, req, serialVersion) {
         this.writeOpCode(dw, OpCode.WRITE_MULTIPLE, serialVersion);
         this.serializeRequest(dw, req);
-        dw.writeString(req.tableName);
+        dw.writeString(req.tableName != null ?
```

Comment:
The other drivers check to see if all operation table names are the same, and if so then just write the one unique table name. Is there code somewhere else that checks this?

Reply:
No need for this, since they can use it only in 2 ways, either specify req.tableName or specify table name for each operation separately.  Specifying both will be an error.  Technically, yeah, they can go against recommendation in the api doc and  specify table name for each operation which is the same for all ops, in which case we would join them with ","s.  But I don't suppose this would cause any problem, right?

Reply:
Note that now writeMany() has tableName as optional argument.  In other languages this would not be allowed as optional arguments must come after required ones.  In JS though I see this is used sometimes, even some Node.js API do it this way.  Note that code in writeMany() in nosql_client.js that checks the arguments.
The problem is that JS does not have method overloads and I didn't think it would be a good idea to create yet another API for this.  However, typescript does have overloads, so when we write typescript declarations for this we can declare 2 overloads for each case (with/without tableName argument).